pkgs/applications/networking/mailreaders/thunderbird/sequoia.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, thunderbird, sequoia-octopus-librnp }:
+let
+  originalPackage = thunderbird;
+
+  # We use `overrideAttrs` instead of defining a new `mkDerivation` to keep
+  # the original package's `output`, `passthru`, and so on.
+  thunderbird-sequoia = originalPackage.overrideAttrs (old: rec {
+    pname = "thunderbird-sequoia";
+    version = "${old.version}+${sequoia-octopus-librnp.version}";
+
+    meta = with lib; {
+      description = "Thunderbird with the Sequoia OpenPGP backend";
+      homepage = "https://gitlab.com/sequoia-pgp/sequoia-octopus-librnp";
+      license = old.meta.license;
+      maintainers = with maintainers; [ puzzlewolf ];
+    };
+
+    propagatedBuildInputs = [
+      sequoia-octopus-librnp
+    ];
+
+    buildCommand = ''
```

Comment:
I'm still not very sure if using `buildCommand` is the best idea. The original Thunderbird basically overrides Firefox expression which uses `stdenv.mkDerivation` builder. I assume that overriding `buildCommand` of that builder isn't recommended by the manual. What can be done is overriding `postFixup`. An advantage would be cleaner override behavior (one wouldn't need to override `thunderbird` and pass it to the `thunderbird-sequoia`, but do it directly). The only downside I see is the need to rebuild Thunderbird, but only until the package lands in the nixpkgs binary cache. Anyway, the need to use `buildCommand` should be cleanly explained here.